app/models/user.rb

```diff
@@ -1,9 +1,12 @@
 class User < ApplicationRecord
-  before_save{self.email = email.downcase}
-  validates :name, presence: true, length: {maximum: Settings.user.name_maxlength}
+  before_save{self.email = email.downcase!}
```

Comment:
tào lao
chị nói thay toàn bộ `self.email = email.downcase!` thành `email.downcase!`á